project/CodeCoverageSettings.scala

```diff
@@ -5,19 +5,12 @@ object CodeCoverageSettings {
 
   private val excludedPackages: Seq[String] = Seq(
     "<empty>",
-    "Reverse.*",
-    "uk.gov.hmrc.BuildInfo",
-    "app.*",
-    "prod.*",
-    ".*Routes.*",
-    "testOnly.*",
-    "testOnlyDoNotUseInAppConf.*",
-    ".*InternalAuthTokenInitialiser.*"
+    ".*Routes.*"
   )
 
   val settings: Seq[Setting[?]] = Seq(
     ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
-    ScoverageKeys.coverageMinimumStmtTotal := 86,
+    ScoverageKeys.coverageMinimumStmtTotal := 57,
```

Comment:
cheers, was kinda confused why it was dropping so drastcially. Anyways, moved the generators to the IT package to make it easier to find the right generators IT or unit tests. Only because the IT tests now being in a separate project now has been configured to depend on the microservice's AppDependencies. Other services point directly to compile the unit test dependencies with test -> test. However doing so caused Google guice issues so I decided this split not have the ITs inherit from unit tests. In my mind it helps keep them cleaner. We can always append separate dependencies to the ITs if ever needed which is almost never.

Reply:
also added back InternalAuthTokenInitialiser to be excluded from code coverage to get back to the 86%, can always create a new ticket to cleanup/add more tests.

Reply:
Yeh i think we need to add `MongoSupport` to the ITs and also need to reorganise some of the app building in the ITs to fix the `test -> test` possibility. We will probably move `hmrc-mongo-test` into separate IT eventually.

For InternalAuthToken, if I remember that codes originates in a library before so probably best to exclude for now.